Packages/com.unity.inputsystem/InputSystem/Editor/Settings/InputSettingsProvider.cs

```diff
@@ -186,6 +189,22 @@ public override void OnGUI(string searchContext)
             }
         }
 
+        private void CustomUpdateModeHelpBox()
+        {
+            var message =
+                "This is not recommended, the default update mode  is dynamic update and should only be changed for compelling reasons.\nPlease refer to the documentation.";
+            Uri link = new Uri("https://docs.unity3d.com/Packages/com.unity.inputsystem@1.13/manual/Settings.html#update-mode");
```

Comment:
Use `InputSystem.kDocUrl` in `AssemblyInfo.cs` to form this string so it can be automatically updated here when the package version is bumped.